### DISCUSSION TITLE
Hajimari PC version .tbl files

### DISCUSSION THREAD

**Discussion by SapphireBreeze** [TwnKey/SenScriptsDecompiler#15] [Q&A]
Will your tool be compatible with the .tbl files of the CLE PC port of Hajimari no Kiseki?
Sorry if the question is silly, I'm not sure if the files are actually encrypted by CLE or just not compatible with the current version of the decompiler.

**Comment by TwnKey** [comment]
Well the Sen Script tool is not supposed to be used on TBL files, only on scripts.

Tbl and script files are indeed encrypted.

Someone told me the client can read unencrypted script files (which start with a 4 bytes signature to tell the client that the file has to be decrypted, so if a file doesnt have that signature, the behaviour is different). 

Thus the JP scripts from the PS4 version can be used, but the current version of the decompiler can't parse them yet. I actually don't have the game nor the time to update the tool right now but in the coming weeks I should update it.

**Reply by SapphireBreeze** [reply]
Thank you for the information and your work on these tools.
